data/pages.yaml

```diff
@@ -5,7 +5,7 @@ ansible_pkg:
 # Content for the index page
 index:
   title: Ansible community documentation
-  intro: Ansible offers powerful and flexible automation that is simple, decentralized, and easy.
+  intro: Ansible offers powerful and flexible automation that is simple, community-powered, and free.
```

Comment:
```suggestion
  intro: Ansible offers open-source automation that is simple, flexible, and powerful.
```

Reply:
I agree with changing the statement but your proposal uses the word "power" twice in the same sentence.

Reply:
Ah, good point, totally agreed (was also thinking about adding `free`), thanks, your suggestion LGTM. Committed, PTAL